### PR TITLE
Profiles: add signal-cli profile

### PR DESCRIPTION
**Pull request by tihadot**
signal-cli: https://github.com/AsamK/signal-cli

Tested on Arch (including AUR package) using command line and dbus interface. I've opted for commenting private-etc out to ensure compatibility with all supported Java configurations. I'm still on the fence about that though, since it should work with private-etc in the majority of environments.

### CODE REVIEW

**Comment by tihadot** [comment]
(please excuse the messy commit history... I'm still learning the finer details of squashing/rebasing)

File: etc/signal-cli.profile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# Description: signal-cli provides a commandline and dbus interface for signalapp/libsignal-service-java
+# This file is overwritten after every install/update
+# Persistent local customizations
+include signal-cli.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+
+noblacklist ${HOME}/.local/share/signal-cli
```
Comment (rusty-snake) [inline etc/signal-cli.profile]: Add `blacklist ${HOME}/.local/share/signal-cli` to `disable-programs.inc`.

File: etc/signal-cli.profile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
```
Comment (rusty-snake) [inline etc/signal-cli.profile]: What about `tracelog`?

File: etc/signal-cli.profile [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+include disable-xdg.inc
+
+mkdir ${HOME}/.local/share/signal-cli
+whitelist ${HOME}/.local/share/signal-cli
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+no3d
```
Comment (rusty-snake) [inline etc/signal-cli.profile]: `nodbus`?

File: etc/signal-cli.profile (hunk 1 above)
Comment (tihadot) [reply etc/signal-cli.profile]: Done

File: etc/signal-cli.profile (hunk 2 above)
Comment (tihadot) [reply etc/signal-cli.profile]: You're right, missed that one

File: etc/signal-cli.profile (hunk 3 above)
Comment (tihadot) [reply etc/signal-cli.profile]: That would break a main feature, the daemon mode via dbus interface

**Comment by tihadot** [comment]
Thank you for your feedback!

File: etc/signal-cli.profile (hunk 3 above)
Comment (rusty-snake) [reply etc/signal-cli.profile]: I over read your 
> and dbus interface

 :upside_down_face:

**Comment by rusty-snake** [comment]
Note about `private-etc` if `private-etc java*` would be supported it would work.

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**